infra/recipes/terraform/azure/sandbox/kubernetes-services.tf

```diff
@@ -0,0 +1,16 @@
+resource "azurerm_kubernetes_cluster" "sandbox" {
```

Comment:
Can we abstract this into infra/recipes/terraform/azure, and for `name`, `dns_prefix` use `variables`?